scrapy/settings/default_settings.py

```diff
@@ -261,6 +261,7 @@
     # Engine side
     'scrapy.spidermiddlewares.httperror.HttpErrorMiddleware': 50,
     'scrapy.spidermiddlewares.offsite.OffsiteMiddleware': 500,
+    'scrapy.spidermiddlewares.stickymeta.StickyMetaParamsMiddleware': 600,
```

Comment:
I haven't read the whole discussion, but how was a priority selected - is it arbitrary, or were there some arguments for it being 600?

Reply:
Hi @kmike it is arbitrary, I was using it on my project and I just used it here. Is there a guideline for these numbers?